.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
pushing vscode ? :D

Reply:
yea....... cause it's kinda neededsee readme notes.

Reply:
will really help developers onboard... those not using vscode i cant help lol